python_modules/dagit/dagit/cli.py

```diff
@@ -61,7 +62,25 @@ def execute_query_from_cli(repository_container, query, variables):
         executor=Executor(),
     )
 
-    str_res = seven.json.dumps(result.to_dict())
+    stack_traces = []
+
+    has_errors = bool(result.errors)
+
+    if has_errors:
+        for error in result.errors:
+            stack_traces.append(get_stack_trace_array(error))
+
+    result_dict = result.to_dict()
+
```

Comment:
just throw a comment in here to describe

Reply:
or pull it out in to a named method `_expand_errors_with_stack_trace` or something